src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+
+public interface OperationStrategy {
+    Map<FruitTransaction.Operation, OperationHandler> operationHashMap = new HashMap<>();
+    FruitTransaction.Operation getOperation(FruitTransaction transaction,
+                                            Map<FruitTransaction.Operation,
+                                                    OperationHandler> operationHashMap);
+
+}
```

Comment:
1. I cannot see any usage of this interface.
2. Please create main method where you will use your components so mentor can see how you intend to use them from reading file till writing report because as is it seems for me that some required logic in the middle is missing
3. Map<FruitTransaction.Operation, OperationHandler> operationHashMap should be argument in method, it should be class variable and passed to constructor